src/explorer-components/PageCard/index.js

```diff
@@ -1,15 +1,14 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import styles from './PageCard.module.scss';
-import { map, includes, each } from 'lodash';
-import { Text, Card } from 'components';
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./PageCard.module.scss";
+import { map, includes, each } from "lodash";
+import { Text, Card } from "components";
 
 const Title = props => {
   const { children, className } = props;
   return (
     <Text
-      className={className}
-      className={styles.title}
+      className={className ? className : styles.title}
```

Comment:
@codeluggage this needs both:
```className={`${className} {styles.title}`}```

Overriding classes don't replace existing classes. They just add to it.  The previous version was bad.... thanks for catching this.